### ISSUE TITLE
Output activation and bi-tempered loss

### ISSUE THREAD

**Issue by shkarupa-alex** [google/bi-tempered-loss#6]
Am i right that with bi-tempered loss last model layer activation should be linear? (not sigmoid or softmax as usual case in keras)

My suggestion comes from:
- https://github.com/google/bi-tempered-loss/blob/master/index.js#L373
- https://github.com/google/bi-tempered-loss/blob/master/tensorflow/loss_test.py#L65 (sigmoid output can't be < 0)

**Comment by eamid** [comment]
Yes, you should view the activations the same way as the "logits" in the last layer that you pass to the cross entropy loss.
